source/principles/index.html.md.erb

```diff
@@ -5,4 +5,4 @@ weight: 20
 
 # <%= current_page.data.title %>
 
-See sidebar to navigate principles.
+See sidebar to navigate principles.
```

Comment:
Was the EOF line break removed by mistake?

Reply:
Good catch thank you